VK/VK/CustomNavController/CustomNavigationController.swift

```diff
@@ -0,0 +1,44 @@
+// CustomNavigationController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Контроллер навигации с кастомными переходами и закрытием.
+final class CustomNavController: UINavigationController, UINavigationControllerDelegate {
+    // MARK: - Public properties
+
+    let interactiveTransition = CustomInteractiveTransition()
+
+    // MARK: - Private life cycle
+
+    override func viewDidLoad() {
+        delegate = self
```

Comment:
```suggestion
        delegate = self
```
я у себя сделал в метод, думаю так нужно